hikari/webhooks.py

```diff
@@ -126,8 +126,8 @@ async def execute(
             interaction webhooks.
 
             Additionally, flags this can only be set for interaction webhooks
-            and the only settable flag is EPHEMERAL; this field is just
-            ignored for non-interaction webhooks.
+            and the only settable flag is EPHEMERAL and SUPPRESS_NOTIFICATIONS;
+            this field is just ignored for non-interaction webhooks.
```

Comment:
I wasn't entirely sure how this line should be worded, but `SUPPRESS_NOTIFICATIONS` does work on a non-interaction webhook.